recipes/lightning-bolts/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "lightning-bolts" %}
+{% set version = "0.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lightning-bolts-{{ version }}.tar.gz
+  sha256: ea65874469e204950837c4998870c87ff1e725668bbb424f8b6d8e91dcda2fdf
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - wheel
+  run:
+    - dataclasses  # [py<=36]
```

Comment:
You can't use selector for noarch packages. You don't have to for `dataclasses`.
The conda package will be empty for python >=3.7. See https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#empty-python-packages